project.clj

```diff
@@ -16,6 +16,7 @@
   :dependencies [[org.clojure/clojure]
 
                  [cheshire]
+                 [org.tcrawley/dynapath "1.0.0"]
```

Comment:
Looks like we have this in clj-parent: https://github.com/puppetlabs/clj-parent/blob/7d37cd237cbbe8e93b461483b1d95c286829800c/project.clj#L91

Should we just bump the version there?

Reply:
I'm not sure, I think maybe Austin told me that there was some existing issue with puppetserver that would break if we did, and the fix hasn't been prioritized yet?  It's currently also pinned in pdb, to fix the same issue, fwiw.